### ISSUE TITLE
Clearing data failing

### ISSUE THREAD

**Issue by ilblackdragon** [nearprotocol/nearcore#2493]
Running a validator on betanet:
```
Apr 17 13:15:04.419  INFO stats: # 2837431 CWuHBVSeo8hLi8GRE9YxUaer2dDaH2HzFhiQhiH46znk V/10 18/18/40 peers  79.3kiB/s  119.0kiB/s   1.20 bps 0 gas/s CPU: 8%, Mem: 1.1 GiB
```
and getting for every block an error `Block on canonical chain shouldn't have refcount 0`:
```
Apr 17 13:13:32.174 ERROR client: Can't clear old data, Error { inner: stack backtrace:
   0: failure::backtrace::Backtrace::new
   1: near_chain::chain::Chain::clear_data
   2: near_client::client::Client::on_block_accepted
   3: near_client::client_actor::ClientActor::process_accepted_blocks
   4: near_client::client_actor::ClientActor::process_block
   5: <near_client::client_actor::ClientActor as actix::handler::Handler<near_network::types::NetworkClientMessages>>::handle
   6: <actix::address::envelope::SyncEnvelopeProxy<A,M> as actix::address::envelope::EnvelopeProxy>::handle
   7: <actix::contextimpl::ContextFut<A,C> as core::future::future::Future>::poll
   8: tokio::task::raw::poll
   9: <std::future::GenFuture<T> as core::future::future::Future>::poll
  10: actix_rt::runtime::Runtime::block_on
  11: near::main
  12: std::rt::lang_start_internal::{{closure}}::{{closure}}
             at src/libstd/rt.rs:52
      std::sys_common::backtrace::__rust_begin_short_backtrace
             at src/libstd/sys_common/backtrace.rs:130
  13: main
  14: __libc_start_main
  15: _start


GC Error: block on canonical chain shouldn't have refcount 0 }
```

**Comment by Kouprin** [comment]
Resolved in #2416

**Comment by bowenwang1996** [comment]
duplicate of #2449
